addons/tagging/functions/fnc_createTag.sqf

```diff
@@ -20,14 +21,20 @@
  * Public: No
  */
 
-params ["_tagPosASL", "_vectorDirAndUp", "_texture", "_object", "_unit", ["_material","",[""]], ["_tagModel", "UserTexture1m_F", [""]]];
+params ["_tagPosASL", "_vectorDirAndUp", "_texture", "_object", "_unit", ["_material","",[""]], ["_tagModel", "UserTexture1m_F", [""]], ["_isVehicleTag", false, [false]]];
 TRACE_5("createTag:",_tagPosASL,_vectorDirAndUp,_texture,_object,_unit);
 
 if (_texture == "") exitWith {
     ERROR_1("%1 is not a valid tag texture.",_texture);
     false
 };
 
+if (_isVehicleTag) exitWith {
+    TRACE_3("tagging vehicle",_object,typeOf _object,_texture);
+    _object setObjectTextureGlobal ["clan", _texture];
```

Comment:
```suggestion
    _object setObjectTextureGlobal [getText (configOf _obj >> "selectionClan"), _texture];
```
In case any mod decides to not name the clan selection "clan".